docs/blog/2019-04-03-how-we-think-about-product-at-gatsby/index.md

```diff
@@ -0,0 +1,130 @@
+
+Typical roadmaps are difficult to use because they don’t communicate information in a way that addresses everyone’s needs.
+
+To make sure that we don’t fall into this trap, we will do outcome-based roadmaps. Our roadmap will include:
+our business vision
+our objectives
+our business results
+
+This will help us focus our roadmaps on business problems we’re trying to solve, rather than a general feature that we’re not sure is important.
+
+Where will all of this information live?
+This is obviously a lot of information to gather and we want to be sure we capture the important parts and provide a reference and source of truth for the team to work from.
+Product Requirement Documents
+Specifically for product, we create Product Requirement Documents (PRDs) that will explain what we want to do and why. They will include information that we need to scope out the product or feature, and also include the evidence that shows why we should do this. Because we want to make sure our decisions are based on actual problems, research, and data we always include that in the PRDs as justification.
+RFCs
+At Gatsby, we also create Request for Comment (RFC) docs. These are detailed proposals that explain what’s changing in the codebase and why. It’s basically a technical proposal for a spec change.
+How do PRDs relate to RFCs?
+PRDs layout what the product, product feature, or product change is and why we are doing it. RFCs relate to what is changing in the codebase and why.
+
+Rolling with changes
+We’re sure that processes and needs will change as we scale and grow. But how we think about product within Gatsby will help us make decisions on whether the processes we currently have are helping us, and give us a foundation for new processes in the future.
+
+## We need your help to be better
+
+You may be wondering why we’re telling you all of this. One of our [company values is to work in the open](https://www.gatsbyjs.org/blog/2018-09-07-gatsby-values/#work-in-the-open). And that means letting our community know what our plans are, how we’re organizing our work, and how you can help us.
+
+This is our initial effort to let you know how we think about product at Gatsby and what it means to us.
+
+We’d love your help in hearing what Gatsby means to you. The first step in all of this is talking with the community and continuing to learn from you.
+
+### Tell us about your experience
+
+One of the best ways you can help is by talking to us about how you use Gatsby now, what works for you, and what problems you have. Getting the opportunity to understand your problems helps us create better solutions for you!
+
+If you’d like to participate in our research efforts, @shannonbux is always looking for research participants. You can [sign up here](https://airtable.com/shrKDSF2E7ljCaYCd) to get notified about upcoming research studies.
```

Comment:
add a link to `@shannonbux` or a name to make it more personal

Reply:
Nice! Here's my twitter profile: https://twitter.com/shannonb_ux

Reply:
Added!